Formula/math-comp.rb

```diff
@@ -0,0 +1,52 @@
+class MathComp < Formula
+  desc "Mathematical Components for the Coq proof assistant"
+  homepage "https://math-comp.github.io/math-comp/"
+  url "https://github.com/math-comp/math-comp/archive/mathcomp-1.6.4.tar.gz"
+  sha256 "c672a4237f708b5f03f1feed9de37f98ef5c331819047e1f71b5762dcd92b262"
+  head "https://github.com/math-comp/math-comp.git"
+
+  depends_on "ocaml" => :build
+  depends_on "coq"
+
+  def install
+    coqbin = "#{Formula["coq"].opt_bin}/"
+    coqlib = "#{lib}/coq/"
+
+    (buildpath/"mathcomp/Makefile.coq.local").write <<~EOS
+      COQLIB=#{coqlib}
+    EOS
+
+    cd "mathcomp" do
+      system "make", "MAKEFLAGS=#{ENV["MAKEFLAGS"]}", "COQBIN=#{coqbin}"
+      system "make", "install", "MAKEFLAGS=#{ENV["MAKEFLAGS"]}", "COQBIN=#{coqbin}"
+
+      elisp.install "ssreflect/pg-ssr.el"
+    end
+
+    if build.head?
+      system "make", "-C", "htmldoc", "COQBIN=#{coqbin}"
```

Comment:
```
      system "make", "-C", "htmldoc", "COQBIN=#{coqbin}" if build.head?
```
is less than or equal to 80 characters so this can be just one line